Akinator.Api.Net.Tests/ClientIntegrationTests.cs

```diff
@@ -351,5 +351,86 @@ public async Task ReuseSessionWorks()
                 Assert.AreEqual(2, question.Step);
             }
         }
+      
+      [TestMethod]
```

Comment:
missing a space

Reply:
I have sorted the test methods alphanumerically. Can you move it infront of the german tests? Also use "Frensh", not "France" in the name. The language is frensh, France is just the country :)